backend/src/main.rs

```diff
@@ -250,10 +263,11 @@ async fn login(
     if let Some(password_hash) = user.password_hash {
         verify_password(&password_hash, &*login_req_opt.password)?;
         let jwt_key = settings.get_jwt_key()?;
+        let secure_cookies = settings.secure_cookies();
         let token = generate_token(&jwt_key, user.id, user.role)?;
 
         let cookie = Cookie::build(("auth_token", token.clone()))
-            .secure(true)
+            .secure(secure_cookies)
```

Comment:
This is not in the scope of this PR. If you need secure cookies to be disabled for development, please move it to a constant and not an environmental variable which defaults to false. I will consider dropping the explicit setting of the secure property separately.

Reply:
I'll remove this before converting the draft PR to ready. This is helpful for testing as secure cookies do not work over http, but I agree false should not be the default.